router/options.go

```diff
@@ -126,6 +126,11 @@ type Options struct {
 	// One use for this option is when your spec defines the entire API of your app but the implementation is spread to multiple microservices.
 	// With this option you can define list of operations that are to be implemented by other microservices.
 	ExcludeOperations []string
+
+	// RuntimeValidateRequestBody defines the behaviour when validating operation response body at runtime. Printing a warning to
+	// the log by default. It is recommended to turn this option to Ignore in production as it can impact performance for large
+	// responses, and to be used in development and testing environments.
+	RuntimeValidateResponseSchema Behaviour `json:"runtimeValidateResponseSchema,omitempty"`
```

Comment:
Maybe rename to `RuntimeValidateResponses`?
I think the validation checks other things and not just the schema like if the content type is correct, etc.

Reply:
I wonder if this should be a global option here or maybe part of the `OperationValidationOptions` struct and not at the root of `Options`.

This way user can define a default behavior for all operations using `DefaultOperationValidation` or override this option for specific operation ids using `OperationValidations`.